avro-converter/src/main/java/io/confluent/connect/avro/AvroConverter.java

```diff
@@ -1,18 +1,16 @@
-/**
- * Copyright 2015 Confluent Inc.
+/*
```

Comment:
Maintain javadoc. I believe this should not be changed to normal comments.

Reply:
There were other places using normal comments and not javadoc, and hence this was used everywhere to make it consistent. I don't believe javadoc matters as Kafka and many other projects use normal comments. 

See https://github.com/google/guava/blob/master/guava/src/com/google/common/base/Absent.java
See https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/clients/admin/AbstractOptions.java#L1